src/jdbc/cubrid/jdbc/net/BrokerHandler.java

```diff
@@ -252,4 +287,50 @@ public static void cancelBrokerEx(String ip, int port, int process, int timeout)
 
         cancelRequest(ip, port, bao.toByteArray(), timeout);
     }
+
+    private static SSLSocket createSSLSocket(Socket plainSocket, String ip, int port) throws UJciException {
+        SSLSocket sslSocket = null;
+        SSLContext ctx = null;
+        SSLSocketFactory sslsocketfactory = null;
+
+        X509TrustManager tm = new X509TrustManager() {
+            public void checkClientTrusted(X509Certificate[] chain, String authType) throws CertificateException {
+            }
+
+            public void checkServerTrusted(X509Certificate[] xcs, String string) throws CertificateException {
+            }
+
+            public X509Certificate[] getAcceptedIssuers() {
+                return new X509Certificate[0];
+            }
+        };
+
+        try {
+            ctx = SSLContext.getInstance("TLS");
+        } catch (NoSuchAlgorithmException e) {
+            e.printStackTrace();
+            throw new UJciException(UErrorCode.ER_CONNECTION, e);
+        }
+
+        try {
+            ctx.init(null, new TrustManager[] { tm }, new SecureRandom());
+        } catch (KeyManagementException e) {
+            e.printStackTrace();
+            throw new UJciException(UErrorCode.ER_CONNECTION, e);
+        }
+
+        sslsocketfactory = ctx.getSocketFactory();
+        try {
+            sslSocket = (SSLSocket) sslsocketfactory.createSocket(plainSocket, ip, port, true);
+            sslSocket.startHandshake();
+        } catch (UnknownHostException e) {
+            e.printStackTrace();
+            throw new UJciException(UErrorCode.ER_CONNECTION, e);
+        } catch (IOException e) {
+            e.printStackTrace();
+            throw new UJciException(UErrorCode.ER_CONNECTION, e);
+        }
```

Comment:
- remove e.printStackTrace();
- only handling IOException is enough. UnknownHostException is extended from IOException.

Reply:
I'll remove the UnknownHostException because I don't need the UnknownHostException code.
however, If We remove e.printStackTrace(), we can't know the correct error about createSocket, startHandshake excption.

Reply:
Then it should be written through Logger class instead of printing it by standard output. please refer getLogger() in UConnection.